src/views/workflowTreeView.ts

```diff
@@ -66,13 +65,7 @@ export class WorkflowTreeView implements vscode.TreeDataProvider<WorkflowTreeEle
     const success = await this.generateWorkflow(element, false);
     if (success) {
       const terminal = this.findOrCreateTerminal();
-      const configVals = ConfigValues.getInstance();
-      terminal.sendText(`export ARTIFICIAL_TOKEN=${configVals.getToken()}`);
-      const customConfigPath: string = vscode.workspace.getConfiguration('artificial.workflow.author').configPath;
-      const fullPath = path.join(this.stubPath, customConfigPath);
-      terminal.sendText(`export ARTIFICIAL_CONFIG=${fullPath}`);
-      const configPath = fullPath.split('config.yaml')[0];
-      terminal.sendText(`(cd ${configPath}; wf publish ${element.path + '.bin'})`);
+      terminal.sendText(`(wf publish ${element.path + '.bin'})`);
```

Comment:
This part depends on an update to wf publish.  Currently I could pass in the path to the config and it will correctly use/overwrite from artificial.env.  But I really shouldn't have to pass in a config path at all IMO.